extractor/extractor/app.py

```diff
@@ -1,15 +1,27 @@
-from extractor import HTTPDownloader, Index, DirectoryCache, MemoryCache, Query, Result
 import io
+from . import HTTPDownloader, Index, DirectoryCache, MemoryCache, Result
 from .filing import Filing
+from .index import IndexRecord
+
+
+def _filter_taxpayer_name(record: IndexRecord) -> bool:
+    # Remove non-profits from MS because it's hard to spell?
+    return "MISSISSIPPI" not in record.taxpayer_name
+
+
+def _filter_website(filing: Filing) -> bool:
```

Comment:
Just curious cuz I don't know rattlesnake about Python, is `_` the standard for prefix for a private method? Is it enforced statically?

Reply:
Yeah, it's just a convention though, it isn't enforced by the language. `__whatever__` is a "magic" method that implements one of the built-in interfaces, so you're not supposed to use that format for your own stuff. Technically, if you prepend the field or method with two underscores the compiler does some kind of munging with the name to make it effectively "private", but doing that isn't all that common AFAIK.